### ISSUE TITLE
[BZ#1901848] Adding information regarding 'Copying Data' phase: Started/Completed/In Progress for each disk

### ISSUE THREAD

**Issue by fdupont-redhat** [konveyor/virt-ui#266]
There could be cases that by looking in Forklift UI it seems as the disk copy was started, but actually it wasn't.
For example, when the PVC for a DataVolume is pending to be bound, the VirtualMachineImport CR is created, but it is not actually copying data. And the MTV UI still displays the Copying Disk state for the virtual machine, leading the using to think it's actually copying.

In the expanded row of the virtual machines tables for a given plan, it would be great that the UI display the state for each individual disk, allowing the user to better understand the state of each DataVolume.

RHBZ: https://bugzilla.redhat.com/show_bug.cgi?id=1901848

**Comment by mturley** [comment]
Assigning myself here because I'm assigned to the BZ and don't want to lose track of this.

**Comment by mturley** [comment]
@vconzola I don't think this is a high priority, but it may be a good thing to knock out if you have time to chat about the design implications some time.

**Comment by vconzola** [comment]
@mturley I'll add it to the list.

**Comment by mturley** [comment]
Hey @vconzola I think we forgot to include this in our plans for the next few weeks, but it's targeted for 2.1. Definitely some design implications to go through here. Can we make time to chat about it?
